### ISSUE TITLE
Request: vnStat as a tab in System Info

### ISSUE THREAD

**Issue by lukyjay** [Hellowlol/HTPC-Manager#355]
Hey,

Title says it all. Is it possible we can have vnStat tables shown inside the System Info page as a tab next to disk and processes? I think this would make sense, make use of the space already available to us and clean up the menu.

Kind Regards,
Jay

**Comment by Hellowlol** [comment]
I agree that having enough space for all the apps are a issue if your running many of the apps supported by htpc manager. The current solution is to shorten the name to prevent the menu from wrapping. Dunno if anybody is using the vnstat module. 

I don't think I will intregrate this in stats module, I'm working on a very (very, very) simple script module. This one will be added next to the smart report tab.

**Comment by brando56894** [comment]
Just to chime in, I use the vnstat module :)

**Comment by Hellowlol** [comment]
Nice, its was a pita getting the dates right, so i glad someone uses it.

**Comment by Hellowlol** [comment]
@brando56894 Would you mind posting a screen shoot of the vnstat module. I would like to update the images in the forum thread and here, but its not that pretty when i dont have any graphs..

**Comment by lukyjay** [comment]
![snapshot2](https://cloud.githubusercontent.com/assets/5250189/10397069/6b4fbfa2-6ed8-11e5-80a9-8d86cfb72f7c.png)
I do not have a large backlog of stats, but hopefully this is of some use to you?

**Comment by Hellowlol** [comment]
Thanks!
